### PR TITLE
webkitRelativePath: Remove 'shortest common ancestor' requirement (Fixes #18)

### LINKED ISSUE #18: Should webkitRelativePath preserve common ancestors?
Spec has:

> In addition, the `webkitRelativePath` property of each File` is set to a relative path from the shortest common ancestor directory of all selected files to that file, including that ancestor in the path. (Therefore this may be a subdirectory of the selected directory rather than the directory itself.)

This is what Chrome does, but (1) none of the other implementations did and (2) we've had bug reports in Chrome about this.  

I'd encourage other implementations to **NOT** follow Chrome here if it's not necessary for web compat, and we should consider changing the spec/fixing Chrome.

### PR DESCRIPTION
**Pull request by inexorabletash**
Not ready for landing - needs test updates.

https://github.com/WICG/entries-api/issues/18

cc: @tkent-google

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -313,10 +313,7 @@ directory rather than a file or files. When specified, the behavior on
 the selection of a directory is as if all files with that directory as
 an ancestor were selected. In addition, the
 {{File/webkitRelativePath}} property of each {{File}} is set to a
-[=relative path=] from the shortest common ancestor directory of all
-selected files to that file, including that ancestor in the path.
-(Therefore this may be a subdirectory of the selected directory
-rather than the directory itself.)
+[=relative path=] from the selected directory to the file.
```
Comment (tkent-google) [inline index.bs]: Edge, Firefox, and Safari set a relative path from the parent of the selected directory.
So, webkitRelativePath includes the basename of the selected directory.

In the following example, the result would be:

- "to_upload/a/b/c/1.txt"
- "to_upload/a/b/c/2.txt"
- "to_upload/a/b/3.txt"

Comment (inexorabletash) [reply index.bs]: Awesome, thanks - will update.

https://github.com/web-platform-tests/wpt/blob/master/entries-api/file-webkitRelativePath-manual.html will need updating. (I should see if there's a way to make the core of those manual tests re-usable as blink layout tests)

Comment (inexorabletash) [reply index.bs]: See latest PS.

File: index.bs [hunk 2]
```diff
@@ -313,7 +313,8 @@ directory rather than a file or files. When specified, the behavior on
 the selection of a directory is as if all files with that directory as
 an ancestor were selected. In addition, the
 {{File/webkitRelativePath}} property of each {{File}} is set to a
-[=relative path=] from the selected directory to the file.
+[=relative path=] starting from (and including) the selected directory
```
Comment (tkent-google) [inline index.bs]: A relative path from selected "/" or "C:\\" might be unclear. If I follow this definition literally, a resultant relative path starts with "/" or "C:\\".  I haven't checked browsers' behavior in this case.

Comment (tkent-google) [reply index.bs]: I confirmed Edge and Firefox produced webkitRelativePath starting with "D:/" if I selected "D:/". So this definition is ok.
I couldn't check Safari behavior due to OOM.

Comment (inexorabletash) [reply index.bs]: Thanks for checking those edge cases!

**Pull request merged**